services/compileGRPCStubs/index.ts

```diff
@@ -13,7 +13,7 @@ const rimraf = require("rimraf");
 const STUBS = "grpc_stubs";
 const GRPC_SUFFIX = "_grpc_pb.js";
 const SERVICE_SUFFIX = "_pb.js";
-const STUBS_FOLDER = "stubs";
+const STUBS_FOLDER_PREFIX = "-grpc-stubs";
```

Comment:
seems like this is SUFFIX. can you rename accordingly.

Reply:
Updated in the latest commit